js/ui/form/ui.form.js

```diff
@@ -1178,11 +1177,8 @@ const Form = Widget.inherit({
         return this.$element().find('.' + FIELD_ITEM_CONTENT_CLASS + ' [tabindex]').first();
     },
 
-    _visibilityChanged: function(visible) {
-        if(visible && browser.msie) {
-            this._refresh();
-        }
-    },
+    // TODO. Deleting this method causes the DataGrid.editing test error.
```

Comment:
Let's not leave todos in PRs. If we need to solve some issue, let's do it now. Nobody will return to it later.